<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>XLex词法自动生成器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icons/resources/analyzser.ico</normaloff>:/icons/resources/analyzser.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>输入正则表达式</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tb_input"/>
      </item>
      <item>
       <widget class="QToolButton" name="btn_clear">
        <property name="icon">
         <iconset theme="edit-clear"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btn_last"/>
      </item>
      <item>
       <widget class="QToolButton" name="btn_next"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_openfile">
        <property name="text">
         <string>打开文件</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btn_toNFA">
        <property name="text">
         <string>生成NFA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_toDFA">
        <property name="text">
         <string>生成DFA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_toMinDFA">
        <property name="text">
         <string>生成MinDFA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_toCode">
        <property name="text">
         <string>生成代码</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableMain"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
